torchvision/models/__init__.py

```diff
@@ -6,3 +6,4 @@
 from .densenet import *
 from .googlenet import *
 from .mobilenet import *
+from .mnasnet import *
```

Comment:
Can you also add an entry in https://github.com/pytorch/vision/blob/master/docs/source/models.rst ?

Reply:
Done, though I'm not sure I've done it correctly. Please take a look.